src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static Colors getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
```

Comment:
Random should be a class field
```suggestion
        int index = random.nextInt(Colors.values().length);
```

Reply:
done